includes/compatibility/lifter/lifter.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ *  On the admin side check if lifters streamline feature is enabled and react accordingly.
+ */
+function enable_streamlined_feature() {
+	$is_lifter_streamnlined_enabled = get_option( 'pmpro_toggle_lifter_streamline_setup' ) == 'true';
+	if($is_lifter_streamnlined_enabled ) {
+		$lifter_streamline =  plugins_url() . '/paid-memberships-pro/css/lifter-streamline.css';
+		wp_register_style( 'pmpro_lifter', $lifter_streamline, [], PMPRO_VERSION, 'screen' );
+		wp_enqueue_style( 'pmpro_lifter' );
+		include_once( ABSPATH . 'wp-admin/includes/plugin-install.php' );
+	} else {
+		wp_dequeue_style( 'pmpro_lifter' );
+	}
+}
+
+add_action( 'admin_init','enable_streamlined_feature' );
+
+/**
+ * On LifterLMS plugin activation, activate streamline feature and install PMPro Courses plugin.
+ */
+function lifter_plugin_activation() {
+	update_option( 'pmpro_toggle_lifter_streamline_setup', 'true' );
+	include_once( PMPRO_DIR . '/classes/' . 'class-quiet-plugin-installer.php' ) ;
+	$slug = 'pmpro-courses';
+	$installer = new Quiet_Skin(compact('title', 'url', 'nonce', 'plugin', 'api') );
+	$installer->download_install_and_activate($slug);
+	exit( wp_redirect("/wp-admin/admin.php?page=pmpro-lifter-streamline") );
```

Comment:
In my testing, this exits before Lifter gets fully activated. Maybe intentional?